app/controllers/api/base_controller.rb

```diff
@@ -1,3 +1,14 @@
 class Api::BaseController < ApplicationController
   layout nil
+
+  # Method to log login attempts
+  def log_login_attempt(user_id:, attempted_at:, successful:, ip_address:)
+    asd = 123
+    LoginAttempt.log_attempt(
```

Comment:
It's important to handle any potential exceptions that could be raised by 'LoginAttempt.log_attempt' to prevent application crashes and to ensure that all login attempts are logged as expected.